.travis.yml

```diff
@@ -3,7 +3,7 @@ language: python
 cache: pip
 
 addons:
-  postgresql: "9.6"
+  postgresql: "11"
```

Comment:
I would stick to 9.6 for all but Odoo 8

Reply:
Isn't it better to test all of them? :thinking:

Reply:
I restored the 9.6 version as default, but still left at least 1 test case for each of the other ones (9.4, 10, 11). Is it OK for you?